### PR TITLE
Send user type on authenticateUser

### PR DESCRIPTION
**Pull request by mustafaboleken**
https://github.com/ant-media/Ant-Media-Server/issues/4295

### CODE REVIEW

File: src/main/java/io/antmedia/console/rest/CommonRestService.java [hunk 1]
```diff
@@ -365,7 +365,7 @@ public Result authenticateUser(User user)
 				session.setAttribute(IS_AUTHENTICATED, true);
 				session.setAttribute(USER_EMAIL, user.getEmail());
 				session.setAttribute(USER_PASSWORD, getMD5Hash(user.getPassword()));
-				message = getDataStore().getUser(user.getEmail()).getScope();
+				message = getDataStore().getUser(user.getEmail()).getScope() + "/" + getDataStore().getUser(user.getEmail()).getUserType();
```
Comment (mekya) [inline src/main/java/io/antmedia/console/rest/CommonRestService.java:368]: Please don't query the database two times at the following line
```
message = getDataStore().getUser(user.getEmail()).getScope() + "/" + getDataStore().getUser(user.getEmail()).getUserType();
```
Do something like that 
```
user = getDataStore().getUser(user.getEmail());
message = user.getScope() + "/" + user.getUserType();
```

**Review by mekya — CHANGES_REQUESTED** [review]
Please don't query the database two times at the same line. Just take a look at the comments.

File: src/main/java/io/antmedia/console/rest/CommonRestService.java (hunk 1 above)
Comment (mustafaboleken) [reply src/main/java/io/antmedia/console/rest/CommonRestService.java:368]: Sure, I will change this line

**Comment by mustafaboleken** [comment]
Note: This pr should be merged with https://github.com/ant-media/Ant-Media-Management-Console/pull/182

**Review by mekya — APPROVED** [review]

**Pull request merged**